tests/accuracy_utils.py

```diff
@@ -26,37 +26,83 @@
 sizes_pow_2 = [2**d for d in range(4, 11, 2)]
 sizes_noalign = [d + 17 for d in sizes_pow_2]
 sizes_1d = sizes_one + sizes_pow_2 + sizes_noalign
-sizes_2d_nc = [1, 16, 64, 1000]
-sizes_2d_nr = [1, 5, 1024]
+sizes_2d_nc = [1] if TO_CPU else [1, 16, 64, 1000]
+sizes_2d_nr = [1] if TO_CPU else [1, 5, 1024]
 
 UT_SHAPES_1D = list((n,) for n in sizes_1d)
 UT_SHAPES_2D = list(itertools.product(sizes_2d_nr, sizes_2d_nc))
-POINTWISE_SHAPES = [(1024, 1024), (16, 1024, 256), (16, 128, 64, 64), (20, 320, 15)]
+POINTWISE_SHAPES = (
+    [(2, 19, 7)]
+    if TO_CPU
+    else [(1,), (1024, 1024), (20, 320, 15), (16, 128, 64, 60), (16, 7, 57, 32, 29)]
```

Comment:
Add a 0-d shape for pointwise operation to test that 0-d works as expected? Although we now flatten them as vectors of shape `(1, )`.

Reply:
> Add a 0-d shape for pointwise operation to test that 0-d works as expected? Although we now flatten them as vectors of shape `(1, )`.

OK. Now there are some bugs when use scalar in tests code. We will fix it soon.